load.environment.php

```diff
@@ -10,6 +10,9 @@
 
 /**
  * Load any .env file. See /.env.example.
+ *
+ * Drupal has no official method for loading environment variables and uses 
+ * getenv() in some places.
  */
-$dotenv = Dotenv::createImmutable(__DIR__);
-$dotenv->safeLoad();
+$dotenv = Dotenv::createUnsafeImmutable(__DIR__);
+$dotenv->load();
```

Comment:
What is the reason you switched `safeLoad` for `load` here? Drush site install now failes because when you didn't create an .env at all.

Reply:
That's just the docs indicating `load()` is the way to do it. I think I need a PR for Drush too.

Reply:
Reading the method and yeah thanks for keeping it safeLoad()